tests/FSharp.Data.Tests/Data/PersonSchema.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Was wondering if this should also covre URIs and any other types in the JSON schema type